app/src/main/java/fr/free/nrw/commons/campaigns/CampaignsPresenter.java

```diff
@@ -108,7 +108,7 @@ public void getCampaigns() {
                     }
 
                     @Override public void onError(Throwable e) {
-                        Log.e(TAG, "could not fetch campaigns: " + e.getMessage());
+                        Timber.tag(TAG).e("could not fetch campaigns: %s", e.getMessage());
```

Comment:
This looks fine, but is it even better to use [`e(Throwable, String, ...)`](https://jakewharton.github.io/timber/timber/log/Timber.html#e-java.lang.Throwable-java.lang.String-java.lang.Object...-)?